doc/Manuals/ig-cs_x-road_6_central_server_installation_guide.md

```diff
@@ -335,7 +338,16 @@ The optional configuration for monitoring parameters is installed by package xr
 
 The central monitoring client may be configured as specified in the [UG-CS](#Ref_UG-CS).
 
-### 2.10 Post-Installation Checks
+### 2.10 Pre-configuration for Registration Web Service
+
+The optional configuration for registration web service is installed by package xroad-centralserver-registration-service. The package is included in the central server installation by default.
+
+Additional registration web service configured are specified in the [UG-SYSPAR](#Ref_UG-SYSPAR) section „Registration service parameters“ and can be defined e.g. in local.ini.
```

Comment:
This could be:

> Configuration parameters for registration web service are specified in the [UG-SYSPAR](#Ref_UG-SYSPAR) section „Registration service parameters“.